modules/pubmaticAnalyticsAdapter.js

```diff
@@ -513,6 +513,8 @@ function executeBidsLoggerCall(e, highestCpmBids) {
     slotsArray.push(slotObject);
     return slotsArray;
   }, []);
+  outputObj.owv = window.PWT?.versionDetails?.openwrap_version || '-1';
+  outputObj.pbv = window.PWT?.versionDetails?.prebid_version || '-1';
```

Comment:
Read Prebid's version from Prebid's global object, so there wont be code change while contributing to vanila prebid.

Reply:
Done